__tests__/commands/openapi/__snapshots__/validate.test.ts.snap

```diff
@@ -172,10 +172,14 @@ ADDITIONAL PROPERTY must NOT have additional properties
   29 |         "summary": "Finds Pets by status",]
 `;
 
-exports[`rdme openapi validate > error handling > should throw an error if an invalid OpenAPI 3.0 definition is supplied 1`] = `[MissingPointerError: Token "Error" does not exist.]`;
+exports[`rdme openapi validate > error handling > should throw an error if an invalid OpenAPI 3.0 definition is supplied 1`] = `
+[ValidationError: API definition schema validation failed.
```

Comment:
Slightly nicer error message here with this header being shown instead of dumping the user out with "Token "Error" does not exist."